ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1675,6 +1675,16 @@ DSPM:
   - high_discharge_air_static_pressure_alarm
   - low_discharge_air_static_pressure_alarm
 
+ZDPC:
```

Comment:
Explain. Differential pressure doesnt usually apply across a zone; thats typically static pressure. What specifically is being measured, and what is the differential across?